src/components/sbb-footer/sbb-footer.tsx

```diff
@@ -7,67 +7,33 @@ import { InterfaceFooterAttributes } from './sbb-footer.custom';
   tag: 'sbb-footer',
 })
 export class SbbFooter {
-  /** Footer appearance */
-  @Prop() public appearance?: InterfaceFooterAttributes['appearance'] = 'primary';
+  /**
+   * Negative coloring variant flag
+   */
+  @Prop() public negative: boolean;
+
```

Comment:
Maybe briefly explain what is the difference between the variants.
```suggestion
/**
* 'default' variant is used...  'clock-columns' instead....
*/
```